eclair-core/src/test/scala/fr/acinq/eclair/db/DbMigrationSpec.scala

```diff
@@ -0,0 +1,112 @@
+package fr.acinq.eclair.db
+
+import akka.actor.ActorSystem
+import com.opentable.db.postgres.embedded.EmbeddedPostgres
+import com.zaxxer.hikari.HikariConfig
+import fr.acinq.eclair.db.Databases.{PostgresDatabases, SqliteDatabases}
+import fr.acinq.eclair.db.migration._
+import fr.acinq.eclair.db.pg.PgUtils.PgLock
+import fr.acinq.eclair.db.pg._
+import org.scalatest.Ignore
+import org.scalatest.funsuite.AnyFunSuite
+import org.sqlite.SQLiteConfig
+
+import java.io.File
+import java.sql.{Connection, DriverManager}
+import java.util.UUID
+import javax.sql.DataSource
+
+@Ignore
+class DbMigrationSpec extends AnyFunSuite {
```

Comment:
nit: can you add a comment to guide users on how to test migration with these tests?

Reply:
Done with 8e8292732. It's pretty basic!